### PR TITLE
Cy 2392 make test resources deploy with blueprints

### PR DESCRIPTION
**Pull request by geokala**

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ git+https://github.com/cloudify-cosmo/cloudify-cli@master#egg=cloudify==master
 git+https://github.com/cloudify-cosmo/cloudify-agent@master#egg=cloudify-agent==master
 mock
 passlib==1.7.1
+git+https://github.com/cloudify-cosmo/cloudify-fabric-plugin@master#egg=cloudify-fabric-plugin==master
```
Comment (tehasdf) [inline test-requirements.txt]: plz pin this and the cli to a specific tag

**Comment by tehasdf** [comment]
this is too big to really review deeply, i mostly went over it for code-level correctness but some logic questions might still be there, but most of the questions are about what was there previously anyway

File: test-requirements.txt (hunk 1 above)
Comment (geokala) [reply test-requirements.txt]: CLI probably shouldn't be pinned, as it (at the moment) needs to be master I think?

Comment (geokala) [reply test-requirements.txt]: Fabric pinned to 1.6.0.

**Pull request merged**